photon-ml/src/main/scala/com/linkedin/photon/ml/diagnostics/hl/PredictedProbabilityVersusObservedFrequencyHistogramBin.scala

```diff
@@ -44,19 +42,19 @@ class PredictedProbabilityVersusObservedFrequencyHistogramBin(
     var observedNegCount: Long = 0L)
   extends Serializable {
 
-  Preconditions.checkArgument(lowerBound >= 0 && lowerBound <= 1.0)
-  Preconditions.checkArgument(upperBound >= 0 && upperBound <= 1.0)
-  Preconditions.checkArgument(lowerBound < upperBound)
-  Preconditions.checkArgument(observedNegCount >= 0L)
-  Preconditions.checkArgument(observedPosCount >= 0L)
+  require(lowerBound >= 0 && lowerBound <= 1.0)
+  require(upperBound >= 0 && upperBound <= 1.0)
+  require(lowerBound < upperBound)
+  require(observedNegCount >= 0L)
+  require(observedPosCount >= 0L)
```

Comment:
Nope, I don't think so. A "user" in this sense is a more general user or consumer of the API this class exposes. This is true for almost any case where you do bounds checking for arguments -- you'd never want to elide these checks during build, because it would allow a consumer to call the function (or constructor in this case) with invalid arguments.